pandas/core/tools/datetimes.py

```diff
@@ -772,28 +792,81 @@ def to_datetime(
     -------
     datetime
         If parsing succeeded.
-        Return type depends on input:
-
-        - list-like:
-            - DatetimeIndex, if timezone naive or aware with the same timezone
-            - Index of object dtype, if timezone aware with mixed time offsets
-        - Series: Series of datetime64 dtype
-        - DataFrame: Series of datetime64 dtype
-        - scalar: Timestamp
-
-        In case when it is not possible to return designated types (e.g. when
-        any element of input is before Timestamp.min or after Timestamp.max)
-        return will have datetime.datetime type (or corresponding
-        array/Series).
+        Return type depends on input (types in parenthesis correspond to
+        timezone or out-of-range timestamp handling issues):
```

Comment:
`handling issue` -> `fallback handling`

Reply:
indeed the sentence was clunky. I feel that both aspects (timezone issues + out of range timestamps) need different names. What about :

    `types in parenthesis correspond to fallback in case of timezone issues or out-of-range timestamps`

Reply:
That's a better phrasing. Would replace `issues` with `unsuccessful timezone or out-of-range timestamp parsing`

Reply:
Nice. done.